ush/fill_jinja_template.py

```diff
@@ -186,6 +186,12 @@ def parse_args(argv):
         action="store_true",
         help="Suppress all output",
     )
+    parser.add_argument(
+        "-i",
+        "--ignore_undefined",
+        action="store_true",
+        help="Ignore undefined jinja template variables",
+    )
```

Comment:
Current SRW jinja2 renderer has a strict requirement that all variables be defined. However, to be able to include the workflow from the beginning this needs to be relaxed. Unactivated workflow sections will completely disappear after rendering so no problem there, however undefined variables that are not used will have `None` entry. The workflow still works in this case.

Reply:
I haven't checked but note that it is not used from `generate_workflow.py` so I left the option just in case I need to use it later, not because it is use.